Tests/TestApp/ContentView.swift

```diff
@@ -2,8 +2,11 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        VStack {
+            Text("Hello, world!")
+                .padding()
+            Text("Subtitle")
+        }
```

Comment:
I added `Text("Subtitle")` to have another element in the view in order to verify the behavior of a `ScreenObject` initialized with more than one expected element getter.

The `VStack` that wraps the two `Text` views is required for both of them to fit in the screen. Otherwise, the first one would take up all the space.